homeassistant/components/geniushub/__init__.py

```diff
@@ -32,6 +33,13 @@
 )
 
 
+def make_debug_log_entries(client):
+    """Make any debug log entries as required."""
+    # pylint: disable=protected-access
+    _LOGGER.debug("zones_raw = %s", client.hub._raw_zones)
+    _LOGGER.debug("devices_raw = %s", client.hub._raw_devices)
```

Comment:
We try to avoid double logging. Combine the messages into one log call.

Reply:
Have made it cone call, but added some newlines if that's OK - the JSON is 100s of lines long..